base/shell/explorer/trayntfy.cpp

```diff
@@ -334,15 +334,25 @@ class CTrayNotifyWnd :
                 swpFlags);
         }
 
-        POINT ptPager = { ContentMargin.cxLeftWidth, ContentMargin.cyTopHeight };
+        POINT ptPager;
 
```

Comment:
```suggestion
```

Reply:
Minor optional formatting here and the 2 other comments below.

Reply:
Done and done :+1: